### ISSUE TITLE
Implementation/usage of @headlessui-tailwindcss

### ISSUE THREAD

**Issue by Sandros94** [P4sca1/nuxt-headlessui#16]
How to implement [@headlessui-tailwindcss](https://github.com/tailwindlabs/headlessui/tree/main/packages/%40headlessui-tailwindcss)?

I'm fairly new to nuxt and jamstacks in general, so I'm happy to learn and help with testing.

I've just discovered healdess ui and since I'm using nuxt I've found this repo, but one tool that would be usefull is [@headlessui-tailwind](https://github.com/tailwindlabs/headlessui/tree/main/packages/%40headlessui-tailwindcss). Basically it lets me style based on the state of the component using tailwind's utility classes.

Even if I change `plugins: [require('@headlessui/tailwindcss')],` to `plugins: [require('nuxt-headlessui',)],` inside tailwind.config.js

**Comment by P4sca1** [comment]
Install nuxt-headlessui, @nuxjs/tailwindcss, and @headlessui/tailwindcss as stated in the respective README files. The tailwind css plugin for headless UI works separately from this module. There should not be anything else to do other than requiring the plug-in in your tailwind.config.js.

**Comment by Sandros94** [comment]
I did find I've didn't install nuxt-headlessui as a devdep, but the others are installed correctly as per respective READMEs.

What I've noticed is that whenever @headlessui/tailwindcss is installed I get a PostCSS warning:
```
WARN                                                                                                                     02:49:02
A PostCSS plugin did not pass the `from` option to `postcss.parse`. This may cause imported assets to be incorrectly transformed.
If you've recently added a PostCSS plugin that raised this warning, please contact the package author to fix the issue.
```

So it made me think that I needed PostCSS installed but still didn't work for some reasons...

After a while I've just digged arround and found out that with the face that inside tailwind.conf.js I had:
```
plugins: [
    require('daisyui','@headlessui/tailwindcss',)
],
```
and having daisyui first made headlessui/tailwindcss not work.
So atm Im stuck on using either one of them, but I'm still in the process of migrating away from daisyui so I still need it.

In the end I will just need to install PostCSS for the warning and understand how the `require`attribute inside tailwind.conf.js works.
